<launch>
    <arg name="sim" default="true" />
    <arg name="speed_percentage" default="100" />

    <group if="$(arg sim)">
        <include file="$(find motoman_hc10_moveit_config)/launch/moveit_planning_execution.launch" />
    </group>
    <group unless="$(arg sim)">
        <include file="$(find motoman_hc10_moveit_config)/launch/moveit_planning_execution_box.launch" >
            <arg name="sim" value="$(arg sim)" />
            <arg name="robot_ip" value="$(arg robot_ip)"/>
            <arg name="controller" value="$(arg controller)"/>
        </include>
    </group>

    <node name="move_robot_server" pkg="motoman_hc10_moveit_config" type="move_robot_server.py" args="$(arg speed_percentage)" output="screen"/>
</launch>
